### PR TITLE
adds updating failures instrcutions

### PR DESCRIPTION
**Pull request by kilbergr**
# [Run dev/staging updates for maintenance window](https://ocio-jira.acf.hhs.gov/browse/OHSH-458)

Changes proposed in this pull request:

- Documentation updates

### CODE REVIEW

File: docs/runbooks/maintenance.md [hunk 1]
```diff
@@ -236,6 +238,25 @@ To fix this run:
 
 `sudo systemctl enable php-fpm.service`
 
+## Updating failures
+
+Sometimes when running the [`update-dev` ansible playbook](https://github.com/OHS-Hosting-Infrastructure/environment-configuration/blob/main/docs/runbooks/how-to-monthly-maintenance.md#update-os-packages-on-dev-and-staging), you'll get a failure. The error may look like this:
+
+```sh
+"Error: Package: 1:v8-devel-3.14.5.10-25.el7.x86_64 (@epel)\n           Requires: v8 = 1:3.14.5.10-25.el7\n           Removing: 1:v8-3.14.5.10-25.el7.x86_64 (@epel)\n               v8 = 1:3.14.5.10-25.el7\n           Obsoleted By: 1:nodejs-libs-16.13.2-7.el7.x86_64 (epel)\n               v8 = 2:9.4.146.24-7.el7
```
Comment (rpdelaney) [inline docs/runbooks/maintenance.md:246]:
```suggestion
"Error: Package: 1:v8-devel-3.14.5.10-25.el7.x86_64 (@epel)
           Requires: v8 = 1:3.14.5.10-25.el7
           Removing: 1:v8-3.14.5.10-25.el7.x86_64 (@epel)
               v8 = 1:3.14.5.10-25.el7
           Obsoleted By: 1:nodejs-libs-16.13.2-7.el7.x86_64 (epel)
               v8 = 2:9.4.146.24-7.el7
```

File: docs/runbooks/maintenance.md [hunk 2]
```diff
@@ -211,15 +213,15 @@ Before replacing the php version, save backups of `/etc/php.ini` and`/etc/php-fp
...
     `rm www.conf.default`
 
 Add back the php configuration from the backups you saved previously.
 
 1. Restart various services:
-    `service httpd restart`  
-    `sudo systemctl restart php-fpm.service`  
+    `service httpd restart`
+    `sudo systemctl restart php-fpm.service`
     `sudo systemctl restart nginx.service`.
```
Comment (rpdelaney) [inline docs/runbooks/maintenance.md:225]:
````suggestion
```console
service httpd restart
sudo systemctl restart php-fpm.service
sudo systemctl restart nginx.service
```
````

**Review by rpdelaney — APPROVED** [review]
Just markdown pedantry from me

**Comment by kilbergr** [comment]
@rpdelaney I took some of your suggestions, but others interfere with md numbering so I ignored those.

**Pull request merged**